text_formatter.go

```diff
@@ -49,9 +50,16 @@ type TextFormatter struct {
 	// be desired.
 	DisableSorting bool
 
+	// QuoteEmptyFields will wrap empty fields in quotes if true
+	QuoteEmptyFields bool
+
+	// QuoteRune can be set to override the default quote style
+	QuoteRune rune
```

Comment:
I think it'd be completely clear what this means if the description was:

"QuoteRune can be set to the override the default quoting `"` with e.g. `'`."

Should we consider making this a string instead of a rune to not limit people to a single character? That would be hard to fix without breaking backwards compatibility later.